### PR TITLE
Inherit multiple profiles

### PR DESCRIPTION
**Pull request by Felixoid**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- New Feature

Changelog entry (up to few sentences, required except for Non-significant/Documentation categories):

Implement multiple inheritances for user's profiles

Detailed description (optional):

It was impossible to combine multiple profiles in one to construct a complex profile. This 
PR implements the feature to define something like:

```
<yandex>
  <profiles>                     
    <analytics>              
      <profile>humans</profile> 
      <profile>limited</profile>
      <profile>readonly</profile>
  </profiles>
</yandex>
```

### CODE REVIEW

**Comment by Felixoid** [comment]
Here are proper tests still missing, I'll implement them later

File: dbms/tests/users.d/multiprofile.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<yandex>
+  <profiles>
+    <profile_1>
+      <max_parallel_replicas>2</max_parallel_replicas>
+    </profile_1>
+    <profile_2>
+      <max_network_bytes>1234567890</max_network_bytes>
+    </profile_2>
+    <profile_3>
+      <max_insert_block_size>654321</max_insert_block_size>
```
Comment (vitlibar) [inline dbms/tests/users.d/multiprofile.xml]: The settings constraints are also defined in profiles, so I suppose we should collect constraints from all inherited profiles.

**Comment by alexey-milovidov** [comment]
@alesapin Tests are Ok but new config for tests requires Docker image. What is the proper sequence to merge this?

**Comment by Felixoid** [comment]
I'm implementing integration tests here

File: dbms/tests/users.d/multiprofile.xml (hunk 1 above)
Comment (Felixoid) [reply dbms/tests/users.d/multiprofile.xml]: Done in the latest commits

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**